src/node/desktop/src/main/utils.ts

```diff
@@ -188,14 +202,22 @@ function rsessionNotFoundError(): Error {
  * @returns Paths to config file, rsession, and desktop scripts.
  */
 export function findComponents(): [FilePath, FilePath, FilePath] {
+
   // determine paths to config file, rsession, and desktop scripts
   let confPath: FilePath = new FilePath();
   let sessionPath: FilePath = new FilePath();
 
-  const binRoot = new FilePath(app.getAppPath());
   if (app.isPackaged) {
+
+    // resolve session path
+    if (process.platform === 'darwin') {
```

Comment:
Rather than make this change, we should make sure that we package things into the expected Electron application folder (`Resources/app`; double-check the "real" location)

Reply:
Do you remember that I said yesterday that if the app is the binary, `app.getAppPath()` will resolve to `Resources/app` and if we are running the app in debug / live reload mode for example, it will resolve to another location? (`src/node/desktop`). 

Altough I haven't ever seen `rsession` inside `src/node/desktop`, maybe this block of code aims on not forcing the code to find `rsession` inside `src/node/desktop` when we are running the app in debug / live reload mode

Reply:
Thanks! I recall that now. I'll try and use that approach to produce a package with the Electron-expected layout here.